src/components/PageBody/TrackPageViewConditional.tsx

```diff
@@ -0,0 +1,34 @@
+import { headers } from 'next/headers';
+
+import { SiteContentPointer } from '@/lib/api';
+
+import { TrackPageView } from './TrackPageView';
+
+/**
+ * Track page view in GitBook analytics if not disabled
+ */
+export function TrackPageViewConditional(props: {
```

Comment:
It seems a bit too much to have a component just for this instead of javing the condition in a function in `PageBody.tsx`, it's a bit of an indirection